protoc-gen-swagger/genswagger/atlas_patch.go

```diff
@@ -267,6 +272,9 @@ The service-defined string used to identify a page of resources. A null value in
 	if !withPrivateMethods {
 		sw.Definitions = filterDefinitions()
 	}
+
+	sw.Definitions = applyCustomAnnotations(sw.Definitions)
```

Comment:
could you please make a dedicated flag for it with default to true? just in case if someone kh-kh would like to avoid using it.

Reply:
Ok, will do